skills/mindroom-docs/references/page__deployment__google-services-oauth__index.md

```
@@ -30,32 +30,35 @@ http://localhost:8765/api/oauth/google_gmail/callback
+Use `google_oauth_client` when one shared Google Cloud OAuth client should apply to every Google provider.
+Provider-specific services win over `google_oauth_client`.
 
-GOOGLE_SHEETS_CLIENT_ID=...
-GOOGLE_SHEETS_CLIENT_SECRET=...
-GOOGLE_SHEETS_REDIRECT_URI=https://mindroom.example.com/api/oauth/google_sheets/callback
+Store these fields on the client config service:
 
-GOOGLE_GMAIL_CLIENT_ID=...
-GOOGLE_GMAIL_CLIENT_SECRET=...
-GOOGLE_GMAIL_REDIRECT_URI=https://mindroom.example.com/api/oauth/google_gmail/callback
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)